### PR TITLE
Update Makefile

### PR DESCRIPTION
**Pull request by petedyer11**
Add .docker-image in the dependency list for conf to make sure it is built before the docker-run command.

### CODE REVIEW

**Comment by JanneKiiskila** [comment]
@ygoyal18  @x85446  please review

File: Makefile [hunk 1]
```diff
@@ -76,7 +76,7 @@ clean: bitbake-clean
 	rm -f .docker-image
 
 .PHONY: conf
-conf: ${POKY}/build/conf/local.conf ${POKY}/build/conf/bblayers.conf
+conf: .docker-image ${POKY}/build/conf/local.conf ${POKY}/build/conf/bblayers.conf
```
Comment (costanic) [inline Makefile]: Since this target doesn't use docker_run, perhaps .docker-image should be added to the `${POKY}/build/conf/%:` target below instead?

**Review by costanic — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (petedyer11) [reply Makefile]: I've moved the .docker-image as suggested and re-tested. Both ways have the same end effect of building the docker image before using it.

**Review by costanic — APPROVED** [review]

**Comment by petedyer11** [comment]
Can  somebody with write access please merge this. Thanks.

**Pull request merged**